cpp/src/barretenberg/honk/pcs/kzg/kzg.test.cpp

```diff
@@ -40,7 +40,8 @@ TYPED_TEST(BilinearAccumulationTest, single)
 
     auto prover_transcript = ProverTranscript<Fr>::init_empty();
 
-    KZG::reduce_prove(this->ck(), opening_pair, witness, prover_transcript);
+    auto quotient_W = KZG::compute_opening_proof_polynomial(opening_pair, witness);
```

Comment:
Really prefer this because I don't have to think about what the name means.

Reply:
You mean you prefer my change or the original?

Reply:
Yes, it requires much less thinking to  what your code is doing IMO